com.umlet.Umlet.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.umlet.Umlet
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+rename-icon: umlet_logo
+rename-desktop-file: umlet.desktop
+rename-appdata-file: metainfo.xml
+command: umlet
+
+finish-args:
+    - --share=ipc
+    - --socket=x11
+    - --device=dri
+    - --filesystem=home
+    - --filesystem=xdg-config/fontconfig:ro
```

Comment:
why is this needed?

Reply:
this is a bit tricky, this is an attempt to make the font not look absolutely terrible, also see https://github.com/flatpak/flatpak/issues/4122

Adding that line, makes the font at least aliased, which is better than nothing I guess.